### ISSUE TITLE
feat: add host account hash in sessionInfo

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1436]
This will allow things like the sticker API to differentiate between two host account numbers with the exact same last 4 digits.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.7.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.6.13...3.7.0) (2021-03-02)

- build(deps): bump patch-package from 6.4.2 to 6.4.4 [`#1435`](https://github.com/open-wa/wa-automate-nodejs/pull/1435)
- build(deps): bump socket.io from 3.1.1 to 3.1.2 [`#1404`](https://github.com/open-wa/wa-automate-nodejs/pull/1404)
- build(deps-dev): bump marked from 2.0.0 to 2.0.1 [`#1423`](https://github.com/open-wa/wa-automate-nodejs/pull/1423)
- build(deps-dev): bump eslint from 7.20.0 to 7.21.0 [`#1422`](https://github.com/open-wa/wa-automate-nodejs/pull/1422)
- build(deps): bump patch-package from 6.2.2 to 6.4.2 [`#1426`](https://github.com/open-wa/wa-automate-nodejs/pull/1426)
- build(deps): bump puppeteer from 7.0.0 to 8.0.0 [`#1408`](https://github.com/open-wa/wa-automate-nodejs/pull/1408)
- build(deps): bump eventemitter2 from 6.4.3 to 6.4.4 [`#1403`](https://github.com/open-wa/wa-automate-nodejs/pull/1403)
- build(deps): bump ts-json-schema-generator from 0.86.1 to 0.88.0 [`#1421`](https://github.com/open-wa/wa-automate-nodejs/pull/1421)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.15.2 to 4.16.1 [`#1427`](https://github.com/open-wa/wa-automate-nodejs/pull/1427)
- build(deps-dev): bump @typescript-eslint/parser from 4.15.2 to 4.16.1 [`#1428`](https://github.com/open-wa/wa-automate-nodejs/pull/1428)
- build(deps-dev): bump husky from 5.0.9 to 5.1.3 [`#1429`](https://github.com/open-wa/wa-automate-nodejs/pull/1429)
- 🐛  fixes #1437 [`#1437`](https://github.com/open-wa/wa-automate-nodejs/issues/1437)
- 🐛  fixes #1432 [`#1432`](https://github.com/open-wa/wa-automate-nodejs/issues/1432)
- Release 3.7.0 [`0813989`](https://github.com/open-wa/wa-automate-nodejs/commit/0813989145488b94705b5d613825025a8df1eea5)
- 🚨  added eslint [`20d99f6`](https://github.com/open-wa/wa-automate-nodejs/commit/20d99f6d77a6a901ee14cfe95efdf19618975387)
- 🚨 postman builder now passing eslint [`cbcb282`](https://github.com/open-wa/wa-automate-nodejs/commit/cbcb2827817e9fe40460831a310e3187dd0b460c)
- ✨  `client.getLicenseType()` #1431 [`37e7f7a`](https://github.com/open-wa/wa-automate-nodejs/commit/37e7f7ab9e0b3ba40d6af47cca20cd02ea67706e)
- 🔥  removed `typescript` requirement just for a couple of enums [`cefb757`](https://github.com/open-wa/wa-automate-nodejs/commit/cefb757458d856d8e5941decb249b13319aa988e)
- 💚  attempt to fix missing emojis in release image [`4f98211`](https://github.com/open-wa/wa-automate-nodejs/commit/4f982118cc85d708233dbb9a301f8052739f3554)
- 📝  bump wa version in README #1391 [`efda590`](https://github.com/open-wa/wa-automate-nodejs/commit/efda59070f1f4be61d20a8f5d0d1214aacf64e96)
- ✨ `sessionInfo` now includes `NUM_HASH` #1436 [`b492d59`](https://github.com/open-wa/wa-automate-nodejs/commit/b492d597092d96194ad61ecc54d68bc124289c06)
- 🥚 #1434 #1433 #1430 [`7df6906`](https://github.com/open-wa/wa-automate-nodejs/commit/7df6906865b9fdaac6511bfc4071681979ca7702)

**Comment by smashah** [comment]
https://docs.openwa.dev/classes/client.html#getlicensetype
